deps/v8/include/v8-version.h

```diff
@@ -11,7 +11,7 @@
 #define V8_MAJOR_VERSION 6
 #define V8_MINOR_VERSION 6
 #define V8_BUILD_NUMBER 346
-#define V8_PATCH_LEVEL 32
+#define V8_PATCH_LEVEL 33
```

Comment:
Can you undo this and bump `v8_embedder_string` in common.gypi instead?

Reply:
Thanks. Done.